parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -84,19 +80,20 @@
    {
       "location": "[variables('location')]",
       "name": "[variables('{{.Name}}AvailabilitySet')]",
-      "apiVersion": "[variables('apiVersionStorageManagedDisks')]",
+      "apiVersion": "[variables('apiVersionDefault')]",
```

Comment:
move the `apiVersion` change out of the `{{if .IsManagedDisks}}` block now that it's the same for both conditions

Reply:
changed the `apiVersion` back to `apiVersionStorage` for storage account so this isn't true anymore